include/koa/authentication/passport-strategies/strategy_services.js

```diff
@@ -50,10 +51,10 @@ function _getQuery(loginContext, pb, ignorePassword) {
     return query;
 };
 
-async function getUser(loginContext, done, pb, ignorePassword = false) {
+async function getUser(loginContext, identityProvider, done, pb, ignorePassword = false) {
```

Comment:
There was once upon a time a convention for passing in params.  Like PB had a certain place and callback/done was last.  I dont remember nor do I care particularly at this point.